### DISCUSSION TITLE
Black Screen at open

### DISCUSSION THREAD

**Discussion by Sourc3w** [bridge-core/bridge.#309] [Q&A] [answered]
Hi bro! you program is the best, I was using it a lot time ago but now I have try to download again and the program give a black screen. I am not english speaker and Im not GITHUB user so maybe Im doing bad, but I dont understand why this black screen, maybe I need some extension now or IDK. Well, I have write you in youtube too, I wait your answer. REGARDS!

**Comment by Sourc3w** [comment]
https://www.youtube.com/watch?v=LUXTaLD65_A&t=487s with this program I did a lot new enemy, ores and swords, but now Im sad for the black screen

**Comment by solvedDev** [comment, marked as answer]
Have you tried uninstalling & reinstalling bridge. to fix the issue?

_We sadly cannot find a general fix for the issue unless you can provide a bit more detail on what exactly you did before the error happened. We essentially need information on how we reproduce the issue you're having._

**Reply by Sourc3w** [reply]
hey bro, I have installed java and jdk and it work now, IDK if it is for that but work now. The problem was the program was full black, I could click, minimice in the corner but full black. Im so nervous to see the new functions :D is there some method to transform my 1.14 version to the new? some functions dont work now. I dont care that, Im so excited to try new addons! Thanks for your job and sorry for my english.

**Reply by solvedDev** [reply]
Glad to hear that you were able to fix the issue! 
There's currently no automatic updater for your add-ons. You have to manually fix broken functionality
